### PR TITLE
[instruments] Update Test + Help docs

### PR DESCRIPTION
**Pull request by zaliqarosli**
## Brief summary of changes

- [ ] Have you updated related documentation?

#### Testing instructions (if applicable)

1.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)

### CODE REVIEW

**Comment by aarushichaudhry** [comment]
> ## Brief summary of changes
> * [ ]  Have you updated related documentation?
> 
> #### Testing instructions (if applicable)
> 
> #### Link(s) to related issue(s)
> * Resolves #  (Reference the issue this fixes, if any.)

Linking everything that I have tested here: 

1. https://github.com/aces/Loris/projects/28#card-88753425 -tested
2. https://github.com/aces/Loris/projects/28#card-88753430 -tested
3. https://github.com/aces/Loris/projects/28#card-88753435 - tested and found a bug here. I had linked the details of the issue here: https://github.com/aces/Loris/issues/8552
I am not sure how to check the standard boxes (ajax dir, data framework etc.) but I have tested the modules assigned to me and 1. publication and  2. configuration seems to function as we would expect them to. However, there are bugs in the issue tracker that may be critical to deal with before the release.

File: modules/instruments/help/instruments.md
Comment (aarushichaudhry) [inline modules/instruments/help/instruments.md:1]: Regarding 4. If a form has multiple pages, enter and save data on each subpage, then return to the Top page and click "Save".  Scoring will be calculated only after Save is clicked.  After entering all data, use the status flags and "Send to DCC" in the left sidebar to complete the instrument form. 

For most instruments, you don't have to return to the top page to save the data you have inserted on the subpages. We can enter data on each subpage and have the option of saving the data on that page itself. However, if the multiple subpages have data that needs to be scored, we display the final scores on the top page underneath the general descriptive information (birth, sex etc.)

Comment (zaliqarosli) [reply modules/instruments/help/instruments.md:1]: @aarushichaudhry thanks for your review. next time, you can add your comment directly on the line you're referring to by clicking on the '+' button at the beginning of the line.
Screenshot 2023-04-25 at 12 22 12 PM

I had trouble finding what you were commenting on. thanks!

Comment (zaliqarosli) [reply modules/instruments/help/instruments.md:1]: fixed. please re-review!

**Review by aarushichaudhry — APPROVED** [review]

**Pull request merged**